zoomChatDecoderVer2.py

```diff
@@ -0,0 +1,95 @@
+

```

Comment:
A title or time stamp in the result file is a good idea. Also, option to choose the name of new file should be added.